terrawrap/utils/config.py

```diff
@@ -86,6 +89,152 @@ def parse_wrapper_configs(wrapper_config_files: List[str]) -> WrapperConfig:
     return wrapper_config_obj
 
 
+def is_config_directory(directory: str) -> bool:
```

Comment:
Given a dir, checks to see if there are any files ending in .tf, if so it is a config directory